README.markdown

```diff
@@ -195,72 +195,216 @@ int main(int argc, char**argv) {
 
 
 # BUILDING g3log: 
------------
-The default is to build an example binary 'g3log-FATAL-contract' and 'g3log-FATAL-sigsegv'. I suggest you start with that, run it and view the created log also.
-
-If you are interested in the performance or unit tests then you can 
-enable the creation of them in the g3log/CMakeLists.txt file. See that file for 
-more details
+g3log uses CMake as a one-stop solution for configuring, building, installing, packaging and testing on Windows, Linux and OSX.
 
+g3log also needs Git to extracting version information from the history commits of this repository.
 
 ```
+git clone https://github.com/KjellKod/g3log
 cd g3log
-cd 3rdParty/gtest
-unzip gtest-1.7.0.zip
-cd ../../
 mkdir build
 cd build
 ```
-## Configuring for installing on nix (OSX, Linux, MinGW)
-Default install prefix on Linux is `/usr/local`
-To change it please set  `CPACK_PACKAGING_INSTALL_PREFIX `
+## Configuring
+g3log provides following CMake options (and default values):
+```
+$ cmake -LAH

```

Comment:
Super! Didn’t know you could trigger a listing of all the variables. 

nit: explain above the command in a one liner what `cmake -LAH` is

Reply:
It doesn’t hurt to mention `Options.cmake`

Suggestion: something like 
```

List all of g3log’s cmake options to see configuration choices. 

For additional option context and comments  please also see  [Options.cmake](https://github.com/KjellKod/g3log/blob/master/Options.cmake)

*Generate listing of all of g3log’s CMake variables:*
`cmake-LAH`
```

Reply:
One more commit is added. And the second sentence is added after those options